src/main/java/com/waytta/SaltAPIBuilder.java

```diff
@@ -356,7 +359,7 @@ public static FormValidation doTestConnection(
                 @QueryParameter String servername,
                 @QueryParameter String credentialsId,
                 @QueryParameter String authtype,
-        		@AncestorInPath Item project) {            	
+        		@AncestorInPath Item project) throws InterruptedException, IOException {
```

Comment:
It is a form validation method. In the case of these exceptions you will get a pretty bad UI. You would rather want to catch exceptions and return them in `FormValidation.error`